indico/modules/events/templates/management/_contribution_field_list.html

```diff
@@ -4,7 +4,7 @@
             <div class="contribution-field">
                 <span class="field-title">{{ field.title }}</span>
                 {% if field.field_type %}
-                    <span class="field-type">({{ field.field.friendly_name }})</span>
+                    <span class="field-type">({{ field.get_field().friendly_name }})</span>
```

Comment:
Wouldn't it be more consistent to use `management=True` here even though it makes no difference at the moment?